sketches/BLE_Scratch/robot.cpp

```diff
@@ -0,0 +1,50 @@
+#include "robot.h"
+#include <Servo.h>
+
+Robot::Robot(int rightWheelPin, int leftWheelPin) {
+  rightWheel = Servo();
+  leftWheel = Servo();
+  rightWheel.attach(rightWheelPin);
+  leftWheel.attach(leftWheelPin);
+  speed = 10;
+  // used to calibrate the middle point 
+  rightWheel.write(90);
+  leftWheel.write(90);
+}
+
+void Robot::moveForward(int steps) {
+  // TODO: use millis() instead of delay
+  // for loop on steps
+  rightWheel.write(90 + speed);
+  leftWheel.write(90 - speed);
+  delay(1000);
+  rightWheel.write(90);
+  leftWheel.write(90);
+}
+
+void Robot::moveBackward(int steps) {
+  // TODO: use millis() instead of delay
+  // for loop on steps
+  rightWheel.write(90 - speed);
+  leftWheel.write(90 + speed);
+  delay(1000);
+  rightWheel.write(90);
+  leftWheel.write(90);
```

Comment:
probably the best way to stop the servos is to detach them, in the same way it is done in the stop method of the servo.